content/pages/sdks-and-tools/index.md

```diff
@@ -96,6 +62,32 @@ recommend using Java SDK to build apps on Android.
 Refer to [this][android-docs] documentation for more details.
 </Message>
 
+### Next generation SDKs
+
+<Message type='warning'>
+  In mid-August, we will be deprecating the Box Next Gen SDKs in its current form. **Don’t worry, your existing code will continue to work without changes**.
+
+  ## Why we are making this change
+
+  In keeping with industry best practices, we are consolidating the Next Gen and legacy SDKs into a single package. This makes migration efforts much easier and allows you to seamlessly add new capabilities to existing applications still powered by the legacy SDKs. 
+
+  Ultimately, the Next Generation SDKs will replace the current legacy SDKs as the latest major version release. Read more to learn the details.
+
+  To learn more visit our SDK [versioning strategy document][versioning]
```

Comment:
In line 74 you have the sentence "Read more to learn the details" and then in 76 "To learn more visit..." - IMO leave only the second one, it makes more sense :)